packages/playwright-chromium/index.js

```diff
@@ -13,12 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+const path = require('path');
 const {Playwright} = require('playwright-core/lib/server/playwright.js');
 
-module.exports = new Playwright({
-  downloadPath: __dirname,
+const playwright = new Playwright({
   browsers: ['chromium'],
-  respectEnvironmentVariables: true,
 });
+module.exports = playwright;
+
+try {
+  const downloadedBrowsers = require(path.join(__dirname, '.downloaded-browsers.json'));
```

Comment:
why `__dirname` here? This is the default resolve mechanics?
```suggestion
  const downloadedBrowsers = require('./.downloaded-browsers.json');
```